src/atoms/social/speaker/presentational.js

```diff
@@ -4,7 +4,7 @@ import Icon from 'react-simple-icons';
 import {useHover} from 'react-use';
 import ExternalLink from '../../links/external';
 
-const iconSize = 16;
+const iconSize = 24;
```

Comment:
considering [where these are surfaced](https://dsmjs.com/components/?path=/story/molecules-talk--default), does this seem a little big? i agree that it may have been a bit small before, but i was mostly just aligning to the old site.

Reply:
i forgot to mention that the other piece that was somewhat going through my mind is the recommendation to use a [fibonacci-like approach](https://medium.com/codyhouse/create-your-design-system-part-4-spacing-895c9213e2b9#20ff) to sizing when designing things like this. i'm not a designer and didnt put too deep of thought into this for this particular case, but it was at least on my mind. as mentioned, we'll probably give more thought to this if we ever follow through with building out a design system, but thats probably a ways out.